lib/jsdom/living/nodes/Slotable-impl.js

```diff
@@ -0,0 +1,35 @@
+"use strict";
+
+const { findSlot, assignSlot, assignSlotable } = require("../helpers/shadow-dom");
+
+// https://dom.spec.whatwg.org/#mixin-slotable
+// https://dom.spec.whatwg.org/#light-tree-slotables
+class SlotableMixinImpl {
+  _initSlotableMixin() {
+    this._slotableName = "";
+  }
+
+  _attrModifiedSlotableMixin(name, value, oldValue) {
+    if (name === "slot") {
+      if ((!value && oldValue === "") || (value === "" && !oldValue)) {
```

Comment:
As before, don't collapse down the if statements; make the spec correspondence exact.